### DISCUSSION TITLE
Trying to show actions on a Relation Manager

### DISCUSSION THREAD

**Discussion by ghost** [filamentphp/filament#8189] [Help] [answered]
I'm trying to recreate a part from the demo https://demo.filamentphp.com/shop/products/2/edit , where a product has associated comments which are shown as an extra table (Relation Manager) below the edit-form.

How is the authorization for these actions defined because I can only see the 'View' action per row, even though my table in my relation manager class states:
```php
->actions([
    Tables\Actions\ViewAction::make(),
    Tables\Actions\EditAction::make(),
    Tables\Actions\DeleteAction::make(),
])
```

The `Create`  action which is defined inside `->headerActions([...])` is also not shown.

I've tried creating policies with simply `true` authorizations but it still doesn't show up. Only way to make them show is appending `->authorize(true)` as a quick hack.

**Comment by ghost** [comment]
Apparently there is some discrepancy between using the different page types. Im trying to create a simple `/my-resource/123` as a show-page but in that case there appears to happen some different stuff.

- `class MyView extends \Filament\Resources\Pages\ViewRecord` (doesnt show actions properly)
- `class MyView extends \Filament\Resources\Pages\EditRecord` (shows actions properly)

**Reply by danharrin** [reply, marked as answer]
https://filamentphp.com/docs/3.x/panels/resources/relation-managers#read-only-mode

**Comment by ghost** [comment]
Thanks, didn't see that option.
